src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
```

Comment:
Do you need these properties? All elements will be displayed on the new line by default